
<h1 style="color:black;">About</h1>
<p>
I'm a 5th year PhD student at the Icahn School of Medicine at Mount Sinai, Department of Genetics and Genomic Sciences. I'm interested in the human genome architecture and genetic variation, and the role that these play in human traits and different phenotypes.
</p>
<p>
In order to fully understand the role of genetic variation, the first step is identifying all types of genetic variants. My current research is on human diploid genome assembly and structural variation, particularly in complex regions of the human genome. Two complex regions/region-types that I primarily focus on, are tandem repeats which underly several neurological disorders and the immunoglobulin heavy chain locus which encode antibody genes.
</p>
<p>
Questions that I am particularly interested in are: Why are some regions more complex than others? What is the/Is there an evolutionary pressure to maintain these loci more complex? Are there certain biological functions that are enriched for complex regions of the human genome? What is the role of epigenetics in these complex regions? How do these loci influence disease?
</p>

<h1 style="color:black;">Publications</h1>
1.    <u>Rodriguez, O. L.</u> et al. [A novel framework for characterizing genomic haplotype diversity in the human immunoglobulin heavy chain locus.](https://www.biorxiv.org/content/10.1101/2020.04.19.049270v1) <b>bioRxiv</b> doi:10.1101/2020.04.19.049270.
2.    <u>Rodriguez, O. L.</u>, Ritz, A., Sharp, A. J. & Bashir, A. MsPAC: A tool for haplotype-phased structural variant detection. <b>Bioinformatics</b> (2019) doi:10.1093/bioinformatics/btz618.
3.    Libbrecht, M. W.\*, <u>Rodriguez, O. L.</u>\* (\*contributed equally). et al. A unified encyclopedia of human functional DNA elements through fully automated annotation of 164 human cell types. <b>Genome Biology</b>. 20, 180 (2019).
4.    Garg, P. et al. A survey of rare epigenetic variation in 23,116 human genomes identifies disease-relevant epivariations and novel CGG expansions. <b>bioRxiv</b> doi:10.1101/2020.03.25.007864.
5.    Seiden, A. H. et al. Elucidation of de novo small insertion/deletion biology with parent‐of‐origin phasing. <b>Human Mutation</b> vol. 41 800–806 (2020).
6.    Zook, J. M. et al. A robust benchmark for germline structural variant detection. <b>bioRxiv</b> 664623 (2019) doi:10.1101/664623.
7.    Cirelli, K. M. et al. Slow Delivery Immunization Enhances HIV Neutralizing Antibody and Germinal Center Responses via Modulation of Immunodominance. <b>Cell</b> 0, (2019).
8.    Chaisson, M. J. P. et al. Multi-platform discovery of haplotype-resolved structural variation in human genomes. <b>Nature Commun.</b> 10, 1784 (2019).
9.     Moyer, E. et al. MetaNetVar: Pipeline for applying network analysis tools for genomic variants analysis. <b>F1000Res</b>. 5, 674 (2016).
10. TransPac: transposon detection and characterization from long-reads. Submitted.

<h1 style="color:black;">Awards</h1>
1. 2018-2020 F31 Predoctoral Ruth L. Kirschstein National Research Service Award, National Institute of Neurological Disorders and Stroke, National Institute of Health
2. 2017 Mindich Child Health and Development Institute Trainee Pilot Program Grant
3. 2016 American Society of Human Genetics Reviewer's Choice Abstract (abstract scored in top 10% of poster abstracts) and featured in Poster Walk. 
4. Society for the Advancement of Chicanos/Hispanics and Native Americans 2013 National Conference Travel Scholarship
5. University of Washington Genomic Outreach for Minorities Project Travel Scholarship, 2013
6. University of Washington Genomic Outreach for Minorities Project Travel Scholarship, 2014
7. Boston University Bioinformatics Diversity Travel Fellowship, 2014

<h1 style="color:black;">Software</h1>
1. [Pipeline for diploid assembly and structural variant detection](https://github.com/oscarlr/MsPAC)
2. [Framework to resolve the immunoglobulin heavy chain locus](https://github.com/oscarlr/IGenotyper)
